requirements.txt

```diff
@@ -1,15 +1,10 @@
 # Note that these are what's required outside of the genie_python requirements
 h5py
 configobj
-#docker
-#docker-compose
-kafka-python
```

Comment:
`kafka-python` and `ess-streaming-data-types` are used by `BlockServerToKafka` in `ISIS\inst_server` so should stay in requirements.txt